shopfloor/actions/data.py

```diff
@@ -9,6 +9,13 @@
 class DataAction(Component):
     _inherit = "shopfloor.data.action"
 
+    @property
+    def _full_partner_parser(self):
```

Comment:
this is the simpler record parser. Also, "full" should go to the `data_detail` action.